bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/dto/SimpleOnOffModel.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.avmfritz.internal.dto;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.types.State;
+
+/**
+ * See {@link DeviceListModel}.
+ *
+ * @author Joshua Bacher - Initial contribution
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(propOrder = { "state" })
+@XmlRootElement(name = "simpleonoff")
+public class SimpleOnOffModel {
+
+    Boolean state;
+
+    public static State asState(Boolean state) {
+        if (state != null && state == true) {
+            return OnOffType.ON;
+        } else
+            return OnOffType.OFF;
```

Comment:
Please use brackets:

```suggestion
        } else {
            return OnOffType.OFF;
        }
```